database/src/main/java/com/firebase/ui/database/FirebaseIndexArray.java

```diff
@@ -243,10 +244,14 @@ public String toString() {
     /**
      * A ValueEventListener attached to the joined child data.
      */
-    protected class DataRefListener implements ValueEventListener {
+    private final class DataRefListener implements ValueEventListener {
```

Comment:
Missed one!

Reply:
That shouldn't matter, right? We're making it private so it's not mockable anyway.